.github/workflows/cmake-windows.yml

```
@@ -17,50 +17,79 @@ jobs:
+        sed -i.bak 's|--index-url https://download.pytorch.org/whl/cu118||g' post-link.bat
+        cat post-link.sh
         echo "::endgroup::"
         echo "::group::Build"
-        conda build .
+        conda update --all --override-channels -c conda-forge -y
+        conda build . --override-channels -c conda-forge
         echo "::endgroup::"
+        echo $CONDA_PREFIX
+    - name: Test
+      shell: bash -l {0}
+      working-directory: ${{runner.workspace}}
```

actionlint (1.7.4)

property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)